README.md

```diff
@@ -4,9 +4,12 @@ Pulumi's GitHub Actions deploy apps and infrastructure to your cloud of choice,
 and GitHub. This includes previewing, validating, and collaborating on proposed deployments in the context of Pull
 Requests, and triggering deployments or promotions between different environments by merging or directly committing code.
 
+**Note**: This repository no longer contains the code responsible for creating the `pulumi/actions`
```

Comment:
"This repository no longer contains the code responsible for creating the `pulumi/actions` Docker container." 

doesn't quite read right, especially since it implies that the user navigated to the wrong location.

Instead, I'd rephrase as:

> NOTE: This repository contains samples and additional documentation for using Pulumi's Github Actions container. The specific container definition and associate scripts are found
> \[here\].